BaseFilter/sections/foreign.txt

```diff
@@ -966,12 +966,10 @@ news.lent.az##.right_rebans_box > object[width="200"][height="250"]
 az##.adsgarden
 az##.adsbyskyline
 ! https://github.com/AdguardTeam/AdguardFilters/issues/13766
+/^https:\/\/banker\.az\/wp-content\/uploads\/[0-9]{4}\/[0-9]{2}\/[0-9]{3}x[0-9]{3}\./
```

Comment:
Please limit by `$domain` to avoid redundant generic regexp rules.

Reply:
The `banker.az` domain is included in the regexp rule.
Is it not safe to proceed?

Reply:
As I remember, it will be checked on every site.

Reply:
Did you mean that the domain is also used on another website?
I will change it.